keel-core/src/test/kotlin/com/netflix/spinnaker/keel/services/ComparableLinksTests.kt

```diff
@@ -243,24 +243,26 @@ class ComparableLinksTests : JUnit5Minutests {
           repository.constraintStateFor(singleArtifactDeliveryConfig.name, any(), any<String>())
         } answers {
           emptyList()
-          }
-
+        }
 
         every { repository.artifactVersions(releaseArtifact) } returns versions.toArtifactVersions(releaseArtifact)
+      }
 
-        every {
-          repository.getArtifactVersion(any(), any(), any())
-        } answers {
-          PublishedArtifact(arg<DeliveryArtifact>(0).name, arg<DeliveryArtifact>(0).type, arg<String>(1),
-            gitMetadata = GitMetadata(commit = arg<String>(1),
-              commitInfo = Commit(sha = arg<String>(1), link = "stash"),
-              repo = Repo(name = "keel"),
-              project = "spkr"
-            )
-            , buildMetadata = buildMetadata)
-        }
+      context("stash") {
```

Comment:
Github mixed the tests in an unreadable way :( 
did you just moved the existing logic under `stash` and added a new section under `github`?

Reply:
@gal-yardeni Just check the "ignore whitespace" option in the PR to review!

Reply:
yes, I pushed all the existing tests under stash and added a block for git, reusing some existing setup in the `before` block.